draft-ietf-taps-interface.md

```diff
@@ -1726,17 +1722,19 @@ Properties will include different information:
 * Whether the connection can be used to send data. A connection can not be used
   for sending if the connection was created with the Selection Property
   `Direction of Communication` set to `unidirectional receive` or if a Message
-  marked as `Final` was sent over this connection, see {{msg-final}}.
+  marked as `Final` was sent over this connection. See {{msg-final}}.
 
 * Whether the connection can be used to receive data. A connection cannot be
   used for reading if the connection was created with the Selection Property
   `Direction of Communication` set to `unidirectional send` or if a Message
-  marked as `Final` was received, see {{receiving-final-messages}}. The latter
+  marked as `Final` was received. See {{receiving-final-messages}}. The latter
   is only supported by certain transport protocols, e.g., by TCP as half-closed
   connection.
 
-* For Connections that are Establishing: Transport Properties that the
-  application specified on the Preconnection, see {{selection-props}}.
+* For Connections that are Established or Closing: Transport Properties that the
+  application specified on the Preconnection. See {{selection-props}}.
+  Selection properties of type `Preference` will be exposed as boolean values
+  indicating whether or not the property applies to the selected transport.
```

Comment:
I think we should find a better solution for that – will try to make a PR once this one jas landed.